src/metabase/models/table.clj

```diff
@@ -63,7 +63,7 @@
 
 (t2/define-before-delete :model/Table
   [{:keys [db_id schema id]}]
-  (t2/delete! Permissions :object [:like (str "%" (perms/data-perms-path db_id schema id) "%")]))
```

Comment:
this is a meaningful change, but I'll create a new PR for this to make the history clear